### ISSUE TITLE
Intersection type not accepted as upper bound of a generic type parameter

### ISSUE THREAD

**Issue by andrelfpinto** [sjrd/scala-js-ts-importer#67]
Running:

```
sbt 'run simple.ts Simple.scala'
```

With a file `simple.ts`:

```
interface A {}
interface B {}

function f<T extends A & B>(t: T) {
}
```

I get:

```
Parse error at 4.24
'`>'' expected but `&' found
function f<T extends A & B>(t: T) {
                       ^
```

**Comment by sjrd** [comment]
This would be fixed in one sweep with the same fix as for #66, so I'm going to close this as duplicate.

**Comment by andrelfpinto** [comment]
It is exactly what I would say after seeing your response about `typeDesc`.
